models/appointmentModel.js

```diff
@@ -0,0 +1,23 @@
+const mongoose = require('mongoose');
+
+const appointmentSchema = new mongoose.Schema({
+  userid: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'User',
+  },
+  slotid: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'Slot',
+  },
+  PaymentStatus: {
+    type: Boolean,
+    default: false,
+  },
+  appointmentStatus: {
+    type: String,
+    enum: ['waiting', 'attended', 'canceled', 'missed'],
```

Comment:
the waiting is not intuitive, so how about making it "**available**" instead and keeping it the default
here there should be another field called for example (reserved)

Reply:
don't mind this comment. totally ignore it 😂💔